sources/packages/web/src/services/http/EducationProgramApi.ts

```diff
@@ -19,6 +19,19 @@ export class EducationProgramApi extends HttpBaseClient {
     }
   }
 
+  public async getInstitutionForProgram(): Promise<any> {
+    try {
+      const response = await this.apiClient.get(
```

Comment:
Just a suggestion. Instead of using this.apiClient.get we can leverage the getCall() from the base class hence leveraging the benefit of getCall which exists.